exporter-api/src/main/java/io/zeebe/exporter/api/record/value/JobBatchRecordValue.java

```diff
@@ -36,7 +36,7 @@
    *     exceeded, it can happen that the job is handed to another worker and the work is performed
    *     twice.
    */
-  Duration getTimeout();
+  Duration getTimeoutDuration();
```

Comment:
Please revert this change - the plan was not to change the API more the implementations

Reply:
So please rename to `JobBatchRecord#getTimeout` to `#getTimeoutLong` instead, otherwise we have here a breaking API change